bundle/language_en.properties

```diff
@@ -667,6 +672,114 @@ angal.hospital.thevisitinghourvaluesmustbeintherange0to24.msg
 angal.hospital.visitduration.txt                                                                       = Visit duration
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
+angal.inventory.addproduct.label                                                                       = Add products
+angal.inventory.allinventoryrowswithnewlotshouldhaverealqtygreatterthanzero.msg                        = All inventory(ies) row(s) with new lot should have real quantity greater than zero before validation.
+angal.inventory.areyousureyouwantoclosethiswindow.msg                                                  = Are you sure you want to close the window?
+angal.inventory.askforrealquantityempty.msg                                                            = Do you want to print without the real quantity?
+angal.inventory.confirm.btn                                                                            = Confirm
+angal.inventory.confirm.btn.key                                                                        = O
+angal.inventory.confirm.success.msg                                                                    = Inventory confirmation complete. All stock has been accurately verified and updated in the system.
+angal.inventory.cannotcreateanotherinventorywithstatusdraft.msg                                        = Cannot create another inventory with status draft, Because you already have an inventory with status draft or validated.
```

Comment:
```suggestion
angal.inventory.cannotcreateanotherinventorywithstatusdraft.msg                                        = Cannot create another inventory with status draft because there is an inventory with status draft or validated.
```